src/main/java/org/jabref/gui/help/AboutDialogViewModel.java

```diff
@@ -27,14 +27,15 @@
     private static final String LIBRARIES_URL = "https://github.com/JabRef/jabref/blob/master/external-libraries.md";
     private static final String GITHUB_URL = "https://github.com/JabRef/jabref";
     private static final String LICENSE_URL = "https://github.com/JabRef/jabref/blob/master/LICENSE.md";
+    private static final String CONTRIBUTORS_URL = "https://github.com/JabRef/jabref/graphs/contributors";
     private final String changelogUrl;
     private final String versionInfo;
     private final ReadOnlyStringWrapper environmentInfo = new ReadOnlyStringWrapper();
     private final Logger logger = LoggerFactory.getLogger(AboutDialogViewModel.class);
     private final ReadOnlyStringWrapper heading = new ReadOnlyStringWrapper();
-    private final ReadOnlyStringWrapper authors = new ReadOnlyStringWrapper();
-    private final ReadOnlyStringWrapper developers = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper maintainers = new ReadOnlyStringWrapper();
     private final ReadOnlyStringWrapper license = new ReadOnlyStringWrapper();
+    private final ReadOnlyStringWrapper contributors = new ReadOnlyStringWrapper();
```

Comment:
Since it's a static string, there is no need for an observable property. You can simply set it in the fxml via `text = %JabRef...`.